### DISCUSSION TITLE
prompt from image

### DISCUSSION THREAD

**Discussion by sskit** [runew0lf/RuinedFooocus#160] [General]
Hello!
When getting prompt from image the first time the prompt is generated normally, but when loading a new image the new prompt generation does not happen, instead the prompt of the first generation is duplicated. I have to refresh the page. After that it works.
Is it possible to fix this?

**Comment by yownas** [comment]
Hi,
There is a feature where it won't create a prompt if there already is text. This is so that you don't accidentally overwrite a prompt you are working on. Try clearing the prompt and hopefully it will work.

**Reply by sskit** [reply]
Thank you! It all makes sense now. That makes sense.
